filer/admin/folderadmin.py

```diff
@@ -178,17 +185,12 @@ def delete_view(self, request, object_id, extra_context=None):
         except self.model.DoesNotExist:
             parent_folder = None
 
-        admin_context = AdminContext(request)
-        if LTE_DJANGO_1_6:
-            extra_context = extra_context or {}
-            extra_context.update({'is_popup': admin_context.popup})
         if request.POST:
-            # Popup in pick mode. Call super delete view so the objects
-            # actually get deleted. All possible failures in delete_view cause
-            # exceptions, so it is safe to ignore the return value though.
-            super(FolderAdmin, self).delete_view(
-                request=request, object_id=object_id,
```

Comment:
You're probably doing it perfectly, but it doesn't seem right that you're not calling super() anymore.